draft-ietf-ace-cwt-proof-of-possession.xml

```diff
@@ -207,17 +207,13 @@
       </t>
 
       <section title="Confirmation Claim" anchor="Confirmation">
-	<t>
-	  The <spanx style="verb">cnf</spanx> claim is used in the CWT
-	  to contain members used to identify the proof-of-possession key.
-	  Other members of the <spanx style="verb">cnf</spanx> map may be defined because
-	  a proof-of-possession key may not be the only means of confirming
-	  the authenticity of the token.
-	  This is analogous to the SAML 2.0 <xref target="OASIS.saml-core-2.0-os"/>
-	  SubjectConfirmation element in which a number of different
-	  subject confirmation methods can be included
-	  (including proof-of-possession key information).
-	</t>
+	  <t>
+  The <spanx style="verb">cnf</spanx> claim in the CWT is used to carry confirmation methods, some of
```

Comment:
This is a run-on sentence.  Either make it two sentences or change the comma to a semicolon.